### PR TITLE
Take ruma module out of matrix-sdk

### PR DESCRIPTION
**Pull request by bitfriend**
The matrix dev team is splitting `ruma` into `ruma-common`, `ruma-events`, etc.
Also we will use `ruma` instead of `matrix_sdk::ruma`.

### CODE REVIEW

File: native/acter/Cargo.toml [hunk 1]
```diff
@@ -42,17 +42,17 @@ mime2ext = "0.1.52"
...
+ruma-client-api = { workspace = true }
 ruma-common = { workspace = true, features = ["client", "unstable-unspecified"] }
 ruma-events = { workspace = true }
 sanitize-filename-reader-friendly = "2.2.1"
 serde = { version = "1", features = ["derive"] }
-serde_json = "1.0.111"
+serde_json = "1"
 strum = { workspace = true }
-tokio = "1.7"
+tokio = "1"
```
Comment (gnunicorn) [inline native/acter/Cargo.toml:52]:
```suggestion
tokio = "1.7"
```

we need to keep this at the 1.7 minimum.

File: native/media-cache-wrapper/Cargo.toml [hunk 2]
```diff
@@ -10,16 +10,18 @@ rust-version = "1.70"
...
 tracing = "0.1.40"
 
 [dev-dependencies]
+anyhow = "1.0.79"
 matrix-sdk-sqlite = { workspace = true }
 matrix-sdk-test = { workspace = true }
-tokio = "1.35.1"
 tempfile = "3.9.0"
-anyhow = "1.0.79"
+tokio = "1"
```
Comment (gnunicorn) [inline native/media-cache-wrapper/Cargo.toml:26]: we need to keep that at a minimum as well
```suggestion
tokio = "1.35.1"
```

**Review by gnunicorn — APPROVED** [review]
Just two minor requests to keep a minimum bound on tokio

File: native/acter/Cargo.toml (hunk 1 above)
Comment (bitfriend) [reply native/acter/Cargo.toml]: Fixed by https://github.com/acterglobal/a3/pull/1346/commits/cfb67aaf65bfe250bf39a8016ca53eb22c8f3e51

File: native/media-cache-wrapper/Cargo.toml (hunk 2 above)
Comment (bitfriend) [reply native/media-cache-wrapper/Cargo.toml]: Fixed by https://github.com/acterglobal/a3/pull/1346/commits/cfb67aaf65bfe250bf39a8016ca53eb22c8f3e51

**Pull request merged**